libs/designsystem/modal/src/modal/alert/alert.component.scss

```diff
@@ -1,5 +1,8 @@
+@use 'sass:math';
```

Comment:
In the alert with two buttons, when space is constrained and there is not enough room for the text inside the buttons to fit on the same line, break the buttons onto two lines instead of being side-by-side.